dd-sdk-android-gradle-plugin/src/test/kotlin/com/datadog/gradle/plugin/internal/GitRepositoryDetectorTest.kt

```diff
@@ -109,25 +107,42 @@ internal class GitRepositoryDetectorTest {
             ProcessBuilder("git", "init")
                 .directory(tempDir)
                 .start()
-                .waitFor(5, TimeUnit.SECONDS)
+                .waitForSuccess(5, TimeUnit.SECONDS)
         )
         check(
             ProcessBuilder("git", "add", ".")
                 .directory(tempDir)
                 .start()
-                .waitFor(5, TimeUnit.SECONDS)
+                .waitForSuccess(5, TimeUnit.SECONDS)
+        )
+        check(
```

Comment:
so the thing is that CI image has no git user set up, so doing commit there fails. and `waitFor` return `true` only if there is no timeout, it doesn't care about process return value.

Reply:
So this means in the CI it will fail ? The way I see your new `waitForSuccess` function returns : `haveNotTimeout&&exitValue()==0` . The `exitValue` will be `1` for CI right ?

Reply:
no, I fixed the issue related to the run on CI :) it was related to the missing git user (fixed with `git config` calls)

Reply:
Oh I see now...nicely done  👍